packages/expansion-hub/src/internal/ExpansionHub.ts

```diff
@@ -87,8 +88,43 @@ export class ExpansionHubInternal implements ExpansionHub {
         this.nativeRevHub.setResponseTimeoutMs(timeout);
     }
 
-    getADC(): Promise<number> {
-        return this.nativeRevHub.getADC();
+    async getAnalogInput(channel: number): Promise<number> {
+        return await this.nativeRevHub.getADC(channel, 0);
+    }
+
+    async getSystemCurrent(
+        systemType: SystemCurrentType = SystemCurrentType.Battery,
+    ): Promise<number> {
+        switch (systemType) {
+            case SystemCurrentType.I2C:
+                return await this.nativeRevHub.getADC(5, 0);
```

Comment:
These values should be constants that live in RHSPlib, not magic numbers. For the Control Hub, I think we should use names for these, instead of numbers. I'm very open to opinions about that though.